definitions/features/iop_vmaas_database.rb

```diff
@@ -0,0 +1,45 @@
+class Features::IopVmaasDatabase < ForemanMaintain::Feature
+  include ForemanMaintain::Concerns::BaseDatabase
+
+  metadata do
+    label :iop_vmaas_database
+
+    confine do
+      File.exist?('/etc/containers/networks/iop-core-network.json') ||
+        File.exist?('/etc/containers/systemd/iop-core.network')
+    end
+  end
+
+  def configuration
+    @configuration || load_configuration
+  end
+
+  def services
+    [
+      system_service('postgresql', 10, :component => 'iop',
+        :db_feature => feature(:iop_vmaas_database)),
+    ]
+  end
+
+  private
+
+  def load_configuration
+    podman_command = "podman exec iop-service-vmaas-reposcan bash -c 'env |grep POSTGRESQL_'"
```

Comment:
I was trying to think if there was a cleaner way to provide this information.

Reply:
Make the installer store it in a yaml file on the host.

Reply:
We can go this route for now, I will add a re-factoring task to add secrets.